features/messages/impl/src/main/kotlin/io/element/android/features/messages/impl/timeline/components/MessageEventBubble.kt

```diff
@@ -129,15 +137,20 @@ fun MessageEventBubble(
                 .testTag(TestTags.messageBubble)
                 .widthIn(
                     min = MIN_BUBBLE_WIDTH,
-                    max = (constraints.maxWidth * BUBBLE_WIDTH_RATIO).toInt().toDp()
+                    max = (constraints.maxWidth * BUBBLE_WIDTH_RATIO)
+                        .toInt()
+                        .toDp()
                 )
                 .clip(bubbleShape)
                 .combinedClickable(
                     onClick = onClick,
                     onLongClick = onLongClick,
                     indication = ripple(),
                     interactionSource = interactionSource
-                ),
+                )
+                .onGloballyPositioned { coordinates ->
```

Comment:
Using onGloballyPositioned is not really great for performance, also we already have the size in the `drawWithContent` scope.